### ISSUE TITLE
Staging Review finding: Heading structure issues throughout Form 21-22 flow

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#104047]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Accredited Rep Facing (ARF) + Accredited Rep Mgmt (ARM)
**Product:** ARF (Accredited Rep Portal MVP) + ARM (Appoint a Rep 2.0)
**Feature:** Form 21-22: Digital Submission (ARM) and Accept/Decline (ARF)

## Findings details
**VA.gov Experience Standard - issue:** [User can determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

At several points in the flow for finding and appointing an accredited rep, the heading structure doesn't match the content. Screen reader users often use the heading hierarchy to help understand the page and navigate through it quickly, so when the heading structure doesn't match the content users can get lost.

- On [/appoint-rep/introduction/](https://staging.va.gov/get-help-from-accredited-representative/appoint-rep/introduction/) and there's currently an appointed rep, headings jump from an H1 to an H3 with no intermediate H2.
- On [/appoint-rep/representative-select](https://staging.va.gov/get-help-from-accredited-representative/appoint-rep/representative-select) after searching, the "Showing X of Y" message is an H4 but the heading for the result cards are H3s rather than being nested beneath the H4.
- On [/appoint-rep/representative-contact](https://staging.va.gov/get-help-from-accredited-representative/appoint-rep/representative-contact) the "Contact the accredited representative" alert heading is an H2 rather than being nested under the step indicator H2.
- On [/appoint-rep/representative-replace](https://staging.va.gov/get-help-from-accredited-representative/appoint-rep/representative-replace) the headings go back and forth between H3 and H4.
- On [/appoint-rep/authorize-medical](https://staging.va.gov/get-help-from-accredited-representative/appoint-rep/authorize-medical) the accordion trigger is an H2 rather than being nested beneath the H3.
- On [/appoint-rep/authorize-address](https://staging.va.gov/get-help-from-accredited-representative/appoint-rep/authorize-address) the accordion trigger is an H2 rather than being nested beneath the H3.
- On [/appoint-rep/review-and-submit](https://staging.va.gov/get-help-from-accredited-representative/appoint-rep/review-and-submit) the "Our terms and conditions" accordion trigger is an H2 rather than being nested beneath the H3.

## Recommended action

Headings need to have a consistent hierarchical structure that helps describe the content of the page, even if that structure doesn't match the visual presentation of the page. For each of these pages, I recommend you go through an exercise of outlining the content of the page as though it were just a plain text outline on paper (think 9th grade English class). Each H2 should be a big-picture section of the page, each H3 should be a subsection of an H2, each H4 should be a subsubsection under an H3, etc.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by oddball-lindsay** [comment]
We have some duplicate tickets in our backlog so I'm closing this out. You can reference the tickets where we will address this, up in the Issue Description.

**Comment by shiragoodman** [comment]
hey @oddball-lindsay , sorry going to open this back up. We ask that all Staging Review findings tickets remain open until the issue is resolved. We track these tickets, and closing them prematurely will impact our metrics. Please lmk if you have any questions/concerns.  I'm happy to discuss!
